proposed/cache.md

```diff
@@ -193,13 +193,14 @@ interface CacheItemInterface
      * although the method of serialization is left up to the Implementing
      * Library.
      *
-     * Implementing Libraries MAY provide a default TTL if one is not specified.
+     * Implementing Libraries MAY provide a default TTL if none is specified.
```

Comment:
I just checked PSR-7, and "if none" is used consistently there.  So +1 to this adjustment for consistency.